runtime/common/src/assigned_slots/mod.rs

```diff
@@ -376,15 +434,46 @@ pub mod pallet {
 					// Treat failed downgrade as warning .. slot lease has been cleared,
 					// so the parachain will be downgraded anyway by the slots pallet
 					// at the end of the lease period .
-					log::warn!(target: "assigned_slots",
+					log::warn!(
+						target: LOG_TARGET,
 						"Failed to downgrade parachain {:?} at period {:?}: {:?}",
-						id, Self::current_lease_period_index(), err
+						id,
+						Self::current_lease_period_index(),
+						err
 					);
 				}
 			}
 
 			Ok(())
 		}
+
+		/// Assign a Max Permanent Slot.
+		#[pallet::call_index(3)]
+		#[pallet::weight((<T as Config>::WeightInfo::set_max_permanent_slots(), DispatchClass::Operational))]
+		pub fn set_max_permanent_slots(origin: OriginFor<T>, slots: u32) -> DispatchResult {
+			ensure_root(origin)?;
+
+			<MaxPermanentSlots<T>>::put(slots);
+
+			// Emit an event.
+			Self::deposit_event(Event::<T>::MaxPermanentSlotsAssigned { slots });
+			// Return a successful DispatchResult
+			Ok(())
+		}
+
+		/// Assign a max temporary slot number.
+		#[pallet::call_index(4)]
+		#[pallet::weight((<T as Config>::WeightInfo::set_max_temporary_slots(), DispatchClass::Operational))]
+		pub fn set_max_temporary_slots(origin: OriginFor<T>, slots: u32) -> DispatchResult {
+			ensure_root(origin)?;
+
+			<MaxTemporarySlots<T>>::put(slots);
+
+			// Emit an event.
+			Self::deposit_event(Event::<T>::MaxTemporarySlotsAssigned { slots });
+			// Return a successful DispatchResult
```

Comment:
imo the info in these comments is already implicitly obvious from the code making them redundant
```suggestion
			Self::deposit_event(Event::<T>::MaxTemporarySlotsAssigned { slots });
```